{
  Type: Exception,
  Message: The Message,
  StackTrace:
at Tests.MethodThatThrows()
at InnerVerifier.Throws(Action target)
}